torchx/lib/torchx/nif.ex

```diff
@@ -2,8 +2,11 @@ defmodule Torchx.NIF do
   @moduledoc false
   @on_load :__on_load__
 
+  @nif_path Path.join([:code.priv_dir(:torchx), "torchx"])
```

Comment:
The path at compile time may not be the same at runtime (a release for example may move things around, so we need to keep this inside `__on_load__`. Generally speaking, avoid doing things at compile time. :)